### ISSUE TITLE
Add links to the contributors profiles

### ISSUE THREAD

**Issue by adrian-cid** [mglaman/drupalorg-cli#18]
In the contributors section:

> Contributors: (19) sorabh.v6, mglaman, bojanz, heddn, nikathone, jonnyeom, chrisrockwell, chishah92, adanielyan, agoradesign, ctrlADel, Hubbs, lisastreeter, Lukas von Blarer, huzooka, osab, andypost, joachim, robin.ingelbrecht

Would be nice to have links to the user's profile.

**Comment by mglaman** [comment]
If you see https://www.drupal.org/project/commerce/releases/8.x-2.5, they're linked in the individual issue credits. I could have it also link above.

**Comment by adrian-cid** [comment]
Yes I see it, is just to help a little more to find the profile.

**Comment by adrian-cid** [comment]
I say like https://www.drupal.org/project/onlyone/releases/8.x-1.6 I added manually the links to the user's profiles in the Contributors section.
